code/datums/greyscale/greyscale_configs.dm

```diff
@@ -167,3 +167,181 @@
+
+/datum/greyscale_config/badge/rchest/V
+	icon_file = 'icons/mob/modular/badge/V.dmi'
+
+/datum/greyscale_config/badge/rchest/lambda
+	icon_file = 'icons/mob/modular/badge/lambda.dmi'
+
+/datum/greyscale_config/badge/rchest/circle2
+	icon_file = 'icons/mob/modular/badge/circle2.dmi'
+
+/datum/greyscale_config/badge/rarm
+	icon_file = 'icons/mob/modular/badge/shield.dmi'
+	json_config = 'code/datums/greyscale/json_configs/badge/badge_right_arm.json'
+
+/datum/greyscale_config/badge/rarm/triangle
+	icon_file = 'icons/mob/modular/badge/triangle.dmi'
+
+/datum/greyscale_config/badge/rarm/circle
+	icon_file = 'icons/mob/modular/badge/circle.dmi'
+
+/datum/greyscale_config/badge/rarm/square
+	icon_file = 'icons/mob/modular/badge/square.dmi'
+
+/datum/greyscale_config/badge/rarm/V
+	icon_file = 'icons/mob/modular/badge/V.dmi'
+
+/datum/greyscale_config/badge/rarm/lambda
+	icon_file = 'icons/mob/modular/badge/lambda.dmi'
+
+/datum/greyscale_config/badge/rarm/circle2
+	icon_file = 'icons/mob/modular/badge/circle2.dmi'
+
+/datum/greyscale_config/badge/larm
+	icon_file = 'icons/mob/modular/badge/shield.dmi'
+	json_config = 'code/datums/greyscale/json_configs/badge/badge_left_arm.json'
+
+/datum/greyscale_config/badge/larm/triangle
+	icon_file = 'icons/mob/modular/badge/triangle.dmi'
+
+/datum/greyscale_config/badge/larm/circle
+	icon_file = 'icons/mob/modular/badge/circle.dmi'
+
+/datum/greyscale_config/badge/larm/square
+	icon_file = 'icons/mob/modular/badge/square.dmi'
+
+/datum/greyscale_config/badge/larm/V
+	icon_file = 'icons/mob/modular/badge/V.dmi'
+
+/datum/greyscale_config/badge/larm/lambda
+	icon_file = 'icons/mob/modular/badge/lambda.dmi'
+
+/datum/greyscale_config/badge/larm/circle2
+	icon_file = 'icons/mob/modular/badge/circle2.dmi'
+
+/datum/greyscale_config/badge/beretf
+	icon_file = 'icons/mob/modular/badge/shield.dmi'
+	json_config = 'code/datums/greyscale/json_configs/badge/badge_beret_forward.json'
+
+/datum/greyscale_config/badge/beretf/triangle
+	icon_file = 'icons/mob/modular/badge/triangle.dmi'
+
+/datum/greyscale_config/badge/beretf/circle
+	icon_file = 'icons/mob/modular/badge/circle.dmi'
+
+/datum/greyscale_config/badge/beretf/square
+	icon_file = 'icons/mob/modular/badge/square.dmi'
+
+/datum/greyscale_config/badge/beretf/V
+	icon_file = 'icons/mob/modular/badge/V.dmi'
+
+/datum/greyscale_config/badge/beretf/lambda
+	icon_file = 'icons/mob/modular/badge/lambda.dmi'
+
+/datum/greyscale_config/badge/beretf/circle2
+	icon_file = 'icons/mob/modular/badge/circle2.dmi'
+
+/datum/greyscale_config/badge/berets
+	icon_file = 'icons/mob/modular/badge/shield.dmi'
+	json_config = 'code/datums/greyscale/json_configs/badge/badge_beret_side.json'
+
+/datum/greyscale_config/badge/berets/triangle
+	icon_file = 'icons/mob/modular/badge/triangle.dmi'
+
+/datum/greyscale_config/badge/berets/circle
+	icon_file = 'icons/mob/modular/badge/circle.dmi'
+
+/datum/greyscale_config/badge/berets/square
+	icon_file = 'icons/mob/modular/badge/square.dmi'
```

Comment:
You dont have to make new icons ffor all of these you can just change the icon state

Reply:
ok we'll have to talk about greyscale. You obviously know something I dont.